### ISSUE TITLE
axe.write_results('a11y.json', results) failing

### ISSUE THREAD

**Issue by mudgil** [mozilla-services/axe-selenium-python#149]
with open(name, "w", encoding="utf8") as f:
TypeError: expected str, bytes or os.PathLike object, not dict

I upgraded to axe_selenium_python version 2.1.2 from the version 2.1.1, and the error still occurs.

**Comment by kimberlythegeek** [comment]
see #151 

Once I get a review I'll be able to merge, and upload v2.1.3 to pypi
